### PR TITLE
fix ConfirmSendModal tests; optimize other tests

### PR DESCRIPTION
**Pull request by cstenglein**

### CODE REVIEW

File: src/components/Shared/SendModal/SendOnChain/SendOnChain.test.tsx [hunk 1]
```diff
@@ -73,20 +71,20 @@ describe("SendOnChain", () => {
...
-    ) as HTMLInputElement;
+    let addressInput = screen.getByLabelText("wallet.address");
 
     userEvent.clear(addressInput);
     userEvent.type(addressInput, "abc123456789");
 
-    await waitFor(() => expect(addressInput).toHaveClass("input-error"));
+    addressInput = await screen.findByLabelText("wallet.address");
+
+    // expect(addressInput).toHaveClass("input-error");
```
Comment (escapedcat) [inline src/components/Shared/SendModal/SendOnChain/SendOnChain.test.tsx:81]: Should this be removed?

File: src/components/Shared/UnlockModal/UnlockModal.test.tsx [hunk 2]
```diff
@@ -23,11 +27,7 @@ describe("UnlockModal", () => {
   // https://github.com/cstenglein/raspiblitz-web/issues/234
   // skipped due to react v18 update
   test.skip("should enable button if input is not empty", async () => {
```
Comment (escapedcat) [inline src/components/Shared/UnlockModal/UnlockModal.test.tsx:29]: This stays skipped for now?

File: src/components/Shared/UnlockModal/UnlockModal.test.tsx [hunk 3]
```diff
@@ -48,33 +48,25 @@ describe("UnlockModal", () => {
...
-    userEvent.click(await screen.findByText("wallet.unlock"));
+    userEvent.click(screen.getByText("wallet.unlock"));
 
     expect(await screen.findByText("login.invalid_pass")).toBeInTheDocument();
   });
 
   // https://github.com/cstenglein/raspiblitz-web/issues/234
   // skipped due to react v18 update
-  test.skip("should display unlocking text on Unlock", async () => {
+  test.skip("should display unlocking text on unlock", async () => {
```
Comment (escapedcat) [inline src/components/Shared/UnlockModal/UnlockModal.test.tsx:62]: Stays skipped?

**Review by escapedcat — COMMENTED** [review]
All this and no `act` warnings?! Amazing! 👏

File: src/components/Shared/UnlockModal/UnlockModal.test.tsx (hunk 2 above)
Comment (cstenglein) [reply src/components/Shared/UnlockModal/UnlockModal.test.tsx:29]: Yep, couldn't fix it atm.

File: src/components/Shared/UnlockModal/UnlockModal.test.tsx (hunk 3 above)
Comment (cstenglein) [reply src/components/Shared/UnlockModal/UnlockModal.test.tsx:62]: Yep, couldn't fix it atm.

File: src/components/Shared/SendModal/SendOnChain/SendOnChain.test.tsx (hunk 1 above)
Comment (cstenglein) [reply src/components/Shared/SendModal/SendOnChain/SendOnChain.test.tsx:81]: Whoops was not intended, was Just Testing It out

**Review by escapedcat — APPROVED** [review]

**Pull request merged**